_stbt/core.py

```diff
@@ -2196,6 +2196,13 @@ def _log_image_descriptions(
         ))
 
 
+def _iter_frames(depth=1):
```

Comment:
Call it `_iter_stack` to avoid overloading the term "frame" (of video)? And for similarity with the previous implementation -- the caller diff would be:

    - for caller in inspect.stack()[2:]
    + for caller in _iter_stack(depth=2):

I wasn't sure about the name `depth` -- it suggests that you iterate up to a certain depth, rather than starting from a certain depth -- but I see that it's the terminology used by [`sys._getframe`](https://docs.python.org/2/library/sys.html#sys._getframe).